### PR TITLE
Scrapping the top points per game

### PR DESCRIPTION
**Pull request by blakeparkinson**
This was easy to scrap.

Still to do:
1. Insert these results into a new mongo collection. 
2. Make the scrapping more modular to handle multiple stats (maybe loop through urls...not sure)

### CODE REVIEW

File: scripts/top_scores.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                    ppg: ppg,
+                  };
+                  results.push(metadata);
+                })
+                console.log(results);
+
+            })
+        }
+        else{
+          conole.log(error);
```
Comment (kjgross) [inline scripts/top_scores.js]: "console"

**Comment by kjgross** [comment]
looks good so far. Might consider grabbing that player id in the metadata so we can match it out to teams on their site and not need to do matching based on name only.

File: scripts/top_scores.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                  var ppg = $(this).find('td.sort').text();
+                  var metadata = {
+                    name: player,
+                    ppg: ppg,
+                    team: team	
+                  };
+                  results.push(metadata);
+                })
+                _.forEach(results, function(n, key) {
+                  console.log(n.team);
```
Comment (kjgross) [inline scripts/top_scores.js]: @blakeparkinson  This website uses different abbreviations than what we've used elsewhere.

Example: Warriors here is "GS" but we have it as "GSW"

OKC abbrev. is the same, so this works so far for those team members.

**Comment by blakeparkinson** [comment]
@kjgross kuljie. I have it working now. I think next steps are to put the player_models functions it calls into its own model, and also make all of this functionality more modular. So it can take multiple stats, leagues, etc... LMK if u can help me with that

**Comment by kjgross** [comment]
Yea I can do that.  What's is that dogshit fxn crap doing in here?

**Comment by blakeparkinson** [comment]
I dont know. Im removing it

**Pull request merged**